server/src/bot.js

```diff
@@ -2,32 +2,23 @@ import Botkit from 'botkit';
 import dotenv from 'dotenv';
```

Comment:
don't need the .env file to work. Example seen in teamController

Will remove on next pr